### ISSUE TITLE
experiencing same behaviour as #46

### ISSUE THREAD

**Issue by International** [samu/git-diff-details#54]
Hey guys, I've noticed #46 was closed without any info. I'm experiencing the same behaviour. Windows 10, Atom 1.7.0 stable.

**Comment by samu** [comment]
As commented in #46, i don't know how to reproduce it. 

* Did you check the keybindings resolver?
* Do you see any errors?
* Have you tried debugging it yourself? Let me know if you need help debugging it.

**Comment by International** [comment]
I'll try to debug it myself, and post some updates in a few days. Thanks!

**Comment by International** [comment]
I've set some breakpoints in the package, and I get the following error: ```Uncaught TypeError: Cannot read property 'invalidate' of undefined``` at git-diff-details-view:47 . Here's the stack trace:
```
module.exports.AtomGitDiffDetailsView.initialize (git-diff-detail…view.coffee:21)
View (C:\Users\geo\Ap…iews\node…:184)
AtomGitDiffDetailsView (git-diff-details-view.coffee:8)
(anonymous function) (main.coffee:22)
module.exports.Workspace.observeTextEditors (C:\Users\geo\Ap…rkspace.js:307)
module.exports.activate (main.coffee:21)
module.exports.Package.activateNow (C:\Users\geo\Ap…package.js:183)
(anonymous function) (C:\Users\geo\Ap…package.js:156)
module.exports.Package.measure (C:\Users\geo\Ap…\package.js:92)
(anonymous function) (C:\Users\geo\Ap…package.js:149)
module.exports.Package.activate (C:\Users\geo\Ap…package.js:146)
module.exports.PackageManager.activatePackage (C:\Users\geo\Ap…manager.js:538)
(anonymous function) (C:\Users\geo\Ap…manager.js:519)
module.exports.Config.transactAsync (C:\Users\geo\Ap…\config.js:337)
module.exports.PackageManager.activatePackages (C:\Users\geo\Ap…manager.js:514)
module.exports.PackageManager.activate (C:\Users\geo\Ap…manager.js:497)
(anonymous function) (C:\Users\geo\Ap…ronment.js:754)

```

Hope this helps!

**Comment by International** [comment]
I've submitted a pull request. Not sure if @mark-hahn was experiencing the same issue, but for me, that change fixes it.

**Comment by International** [comment]
I can also confirm this bug on a ubuntu 15.04 with atom 1.7.2

**Comment by samu** [comment]
Fixed with #57
